cfg/param_update.yaml

```diff
@@ -0,0 +1,11 @@
+# File types
+file_common: "_common.cpp"
+file_ros: "_ros.cpp"
+file_cfg: ".cfg"
+file_yaml: ".yaml"
+
+# Path to yaml folder
+path_yaml_folder: "/home/srd-rd/catkin_ws/src/ipa_navigation/ipa_navigation_config/config/components"
+
+# Default editor to open files in (command line name of executable)
+editor: "subl"    # sublime
```

Comment:
It is anyway optional to open the files in the editor and as far as I know a lot of the default editors are not really built to open multiple files at once (e.g. nano). Which one should I use as default?

I can also include in the script, that if no editor is set in the config file, the default editor according to `$EDITOR` is used?!